app/shared-components/ToastMessage/ToastMessage.jsx

```diff
@@ -19,24 +19,45 @@ import PropTypes from 'prop-types';
  * @return {JSX} JSX for rendering a Toast Message
  * @memberof SharedComponents
  */
-const ToastMessage = ({ toastText, toastClass, toastExit }) => (
-	<div className="ToastMessage full-width">
-		{toastText && (
-			<div className="callout-container">
-				<div className={`callout toast ${toastClass}`}>
-					<div className="flex-container align-center-middle">
-						<div className="callout-text">
-							{toastText}
+const ToastMessage = ({
+	toastText, toastClass, toastExit, dawnHub
+}) => {
+	// These variables will be used to determine whether the toast should display with Dawn
+	// onboarding styling or the default styling used in GBE
+	const dawnHubClass = dawnHub ? 'dawn-hub' : '';
+	const dawnLayout = dawnHub ? 'align-justify align-middle' : 'align-center-middle';
+
+	let dawnToastText = '';
+	if (dawnHub && toastClass === 'alert') {
+		dawnToastText = 'Error: ';
+	} else if (dawnHub && toastClass === 'success') {
+		dawnToastText = 'Success: ';
+	}
```

Comment:
We should make sure this is i18n-friendly

Reply:
Ah, absolutely. Fixed! Means less lines of code in `ToastMessage.jsx` as well, so win-win, haha.